src/generic/validate_utf32.h

```diff
@@ -71,8 +71,8 @@ simdutf_really_inline result validate_with_errors(const char32_t *input,
       in.swap_bytes();
     }
 
-    const auto too_large = in > standardmax;
-    const auto surrogate = (in + offset) > standardoffsetmax;
+    const auto too_large = in >= standardmax;
```

Comment:
This causes problems with the POWER kernel?

Reply:
Yes, it lacks the `>=` operator, I will fix.

Reply:
Fixed!